stories/card.stories.js

```diff
@@ -71,4 +71,5 @@ stories.add(
   { notes: { markdown: notes } }
 );
 
-export default stories;
+// eslint-disable-next-line import/prefer-default-export
+export { stories };
```

Comment:
Default exports mean something special to storybook with the new config style, so this had to be changed to a named export for re-use in `shot-card.stories.js`.  We don't do this anywhere else.